Vagrantfile

```diff
@@ -23,6 +23,19 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.provision "file", source: File.join(Dir.home, ".gitconfig"), destination: ".gitconfig"
   end
 
+  # Use the boto configuration from the host in the VM, if it is in the
+  # expected location
+  if File.exists?(File.join(Dir.home, ".boto"))
+    config.vm.provision "file", source: File.join(Dir.home, ".boto"), destination: ".boto"
+  end
+
+  if ENV.has_key?('VERSION')
```

Comment:
Randomly copying over environment variables and especially secret credentials to new VMs seems unreasonable if the audience for this Vagrantfile is developers writing patches for Flocker. If we're going down this route we shouldn't be using same Vagrantfile for releases while at the same time recommending it for 3rd party development.

We can either drop mention of Vagrantfile from docs, or have two Vagrantfiles.